web/src/OverviewActionBar.tsx

```diff
@@ -326,6 +411,94 @@ export function FilterRadioButton(props: FilterRadioButtonProps) {
   )
 }
 
+export const FILTER_INPUT_DEBOUNCE = 500 // in ms
+export const FILTER_FIELD_ID = "FilterTermTextInput"
+
+const debounceFilterLogs = debounce((history: History, search: string) => {
+  history.push({ pathname: location.pathname, search })
```

Comment:
Apparently when this happens the textfield loses focus, which seems like it'd be really annoying! If you pause typing for half a second you have to go click in the textfield again to continue typing.

I haven't looked into this too much and don't have much idea what the fix will look like.

Reply:
Nick fixed this issue with #4641. 🎉